xcode/Subconscious/Shared/Components/FirstRun/FirstRunProfileView.swift

```diff
@@ -13,18 +13,34 @@ struct FirstRunProfileView: View {
     @Environment(\.colorScheme) var colorScheme
     
     var body: some View {
+        let did = Did(app.state.sphereIdentity ?? "") ?? Config.default.subconsciousGeistDid
```

Comment:
Same question as above. What does this line do? is Subconscious GeistDid a valid default value?

Reply:
Since we're doing this logic in two places, we may want to consider making it a calculated property of the model.

Reply:
This was just so I could use the SwiftUI preview to develop, but I did introduce a property when I removed it.